### ISSUE TITLE
Links - Color contrast accessibility issue

### ISSUE THREAD

**Issue by daminihs** [wordplaydev/wordplay#673]
### What happened?

![Image](https://github.com/user-attachments/assets/c8031444-ce95-41a2-982a-f090e1e8087f)

Homepage
Issue: Ensure the 3 links are distinguished from surrounding text in a way that does not rely on color.

To solve this problem, you need to fix at least (1) of the following:
-The link has insufficient color contrast of 1.83:1 with the surrounding text. (Minimum contrast is 3:1, link text: #ffb000, surrounding text: #ffffff)
-The link has no styling (such as underline) to distinguish it from the surrounding text

### What browsers are you seeing the problem on?

Chrome

### What operating system are you using?

Windows

**Comment by amyjko** [comment]
Thanks! This one probably needs some visual design thinking to dual encode the link status.
